src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -1184,17 +1174,23 @@ private void onNetworkEnd(
         private void onNetworkSignal(
             SignalFW signal)
         {
-            if (signal.signalId() == CLEANUP_SIGNAL)
-            {
-                cleanup();
-            }
-            else
-            {
-                assert false;
-            }
+            assert signal.signalId() == CLEANUP_SIGNAL;
+            cleanupStreams(signal.traceId(), signal.authorization());
```

Comment:
Please hoist `traceId` and `authorization` parameter values for readability rather than inlining with method call.